app/src/main/res/values/strings.xml

```diff
@@ -30,4 +30,5 @@
 
     <string name="hapticFeedbackKey">io.github.dkter.aaaaa.enableHapticFeedback</string>
     <string name="themeSettingKey">io.github.dkter.aaaaa.themeSetting</string>
+    <string name="uppercase">Caps lock</string>
```

Comment:
minor nit: could you move this to where all the other image button descriptions are (at the top of the file)?

Reply:
done!